deployment/kuadrant/service-account-tokens/05-tier-access.yaml

```
@@ -0,0 +1,52 @@
+  name: inference-gateway-tier-enterprise
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: models-user
+  namespace: llm
+rules:
+  - apiGroups: ["serving.kserve.io"]
+    resources: ["inferenceservices"]
+    # Can be limited to selected models if needed
+    # resourceNames: ["vllm-simulator"] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)